docs/tutorials/projects/guardian.md

```diff
@@ -245,7 +245,7 @@ Then test the components on the [machine's Control tab](/fleet/machines/#control
 
 ## Detect persons and pets
 
-For the guardian to be able to detect living beings, you will use a Machine Learning model from the Viam Registry called [`effdet0`](https://app.viam.com/ml-model/bill/effdet0).
+For the guardian to be able to detect living beings, you will use a Machine Learning model from the Viam Registry called [`EfficientDet-COCO`](https://app.viam.com/ml-model/viam-labs/EfficientDet-COCO).
```

Comment:
Nit: to match caps elsewhere in this PR & docset. Ignore if now is not the time for standardization.

```suggestion
For the guardian to be able to detect living beings, you will use a machine learning model from the Viam registry called [`EfficientDet-COCO`](https://app.viam.com/ml-model/viam-labs/EfficientDet-COCO).
```